src/screens/AvatarBuilder.js

```diff
@@ -92,33 +86,7 @@ const AvatarBuilder = ({ route: { params } }) => {
     />
   ));
 
-  const saveInfo = async (name, color) => {
-    const walletId = selectedWallet.id;
-    const newWallets = {
-      ...wallets,
-      [walletId]: {
-        ...wallets[walletId],
-        addresses: wallets[walletId].addresses.map(singleAddress =>
-          singleAddress.address.toLowerCase() === accountAddress.toLowerCase()
-            ? {
-                ...singleAddress,
-                ...(name && { label: name }),
-                ...(color !== undefined && { color }),
-              }
-            : singleAddress
-        ),
-      },
-    };
-
-    await dispatch(walletsSetSelected(newWallets[walletId]));
-    await dispatch(walletsUpdate(newWallets));
-    updateWebProfile(
-      accountAddress,
-      name,
-      (color !== undefined && colors.avatarBackgrounds[color]) ||
-        currentAccountColor
-    );
-  };
+  const { saveInfo } = useUpdateEmoji();
```

Comment:
nit: can we move this up near the other hooks around line 65 instead of in the middle of our other callback definitions